packages/model/src/gp2/common.ts

```diff
@@ -18,51 +20,10 @@ export type Resource = ResourceNote | ResourceLink;
 
 export const isResourceLink = (resource: Resource): resource is ResourceLink =>
   resource.type === 'Link';
-export const keywords = [
-  'Epidemiology',
-  'Neurology',
-  'Genetics',
-  'Genomics',
-  'Data Science',
-  'GP2 PhD',
-  'Neurodegeneration',
-  'Pharmacogenomics',
-  'Movement Disorders',
-  'Communications',
-  'Patient Advocate',
-  'Machine Learning',
-  'Research Communications',
-  'Patient Engagement',
-  'R',
-  'Bash',
-  'Diversity',
-  'Laboratory Science',
-  'Operations',
-  'Project Management',
-  'Molecular Biology',
-  'Research Grants',
-  'Neurogenetics',
-  'Python',
-  'Biostatistics',
-  'Stata',
-  'Education',
-  'Program Management',
-  'Course Management',
-  'Training',
-  'Biobanking',
-  'Career Development',
-  'Administrative Support',
-  'GP2 Opportunities',
-  "GP2 Master's",
-  'Computer Science',
-  'Outreach',
-  'Neuroimaging',
-  `Parkinson's disease`,
-] as const;
 
 export type Keyword = (typeof keywords)[number];
-export const isKeyword = (data: string | null): data is Keyword =>
-  keywords.includes(data as Keyword);
+export const isKeyword = (data: string | null): data is string =>
+  keywords !== null;
```

Comment:
```suggestion
  data !== null;
```
I believe this was what you meant